matrix/build.gradle

```diff
@@ -25,14 +25,14 @@ def buildProprietaryTestImagesTask = tasks.create("buildProprietaryTestImages")
 def pullProprietaryTestImagesTask = tasks.create("pullProprietaryTestImages")
 
 def proprietaryTargets = [
-    "weblogic" : [
-       "12.1.3" : ["developer"],
-       "12.2.1.4" : ["developer"],
-       "14.1.1.0" : ["developer-8", "developer-11"]
+    "weblogic": [
+        [version: ["12.1.3", "12.2.1.4"], vm: ["hotspot"], jdk: ["8"], args: [tagSuffix: "developer"]],
+        [version: ["14.1.1.0"], vm: ["hotspot"], jdk: ["8"], args: [tagSuffix: "developer-8"]],
+        [version: ["14.1.1.0"], vm: ["hotspot"], jdk: ["11"], args: [tagSuffix: "developer-11"]]
```

Comment:
Can these 2 lines be one? With an array of `jdk`?

Reply:
The reason is that in some cases the `tagSuffix` needs to contain the JDK version and some cases not (the 12. versions). Therefore it cannot be just appended in the Dockerfile and needs to be manually provided, which means the arguments must be different for version 14. where the suffix contains JDK version.